sql/hive/src/test/scala/org/apache/spark/sql/hive/DynamicPartitionPruningHiveScanSuite.scala

```diff
@@ -21,24 +21,17 @@ import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.expressions.{DynamicPruningExpression, Expression}
 import org.apache.spark.sql.execution._
 import org.apache.spark.sql.execution.adaptive.{DisableAdaptiveExecutionSuite, EnableAdaptiveExecutionSuite}
-import org.apache.spark.sql.execution.datasources.v2.BatchScanExec
 import org.apache.spark.sql.hive.execution.HiveTableScanExec
 import org.apache.spark.sql.hive.test.TestHiveSingleton
 import org.apache.spark.sql.test.SQLTestUtils
 
-abstract class DynamicPartitionPruningHiveScanSuite
+abstract class DynamicPartitionPruningHiveScanSuiteBase
```

Comment:
need to update also for `DynamicPartitionPruningHiveScanSuiteAEOff` & `DynamicPartitionPruningHiveScanSuiteAEOn`